tests/appsec/rasp/test_api10.py

```diff
@@ -194,6 +194,7 @@ def setup_api10(self):
             headers={"Content-Type": "application/json"},
         )
 
+    @missing_feature(library="java", reason="TRACE should not have a body: https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.8")
```

Comment:
Wouldn’t it be better to mark it as irrelevant?
If this isn’t something that’s likely to change in upcoming tracer versions, it’s probably best to mark it as irrelevant so we avoid running the test for all Java weblogs unnecessarily.

What do you think? :)

Reply:
Don't really have an opinion, the test is wrong in the sense that TRACE requests cannot have a body, I'll try to fix the test next week and remove the missing_feature from here.